### ISSUE TITLE
Migrate repo to pvlib github org?

### ISSUE THREAD

**Issue by kandersolar** [pvlib-benchmarker/pvlib-benchmarks#4]
@AdamRJensen suggests, and I think I agree, that it would make sense to migrate this benchmarks repository from `pvlib-benchmarker/pvlib-benchmarks` to `pvlib/pvlib-benchmarks`.  I solicit thumbs oriented up or down on this idea.

It seems I can't tag the collective handle from here, so: @wholmgren @cwhanse @adriesse @mikofski @adamrjensen

**Comment by wholmgren** [comment]
@kanderso-nrel can you remind me why we set it up as a separate organization? Was it to preserve the pvlib github.io space?

**Comment by kandersolar** [comment]
If I understand correctly, repo-level GitHub Pages sites go in a subfolder on the main domain, so after moving this repo its pages would appear under `https://pvlib.github.io/pvlib-benchmarks/`.  IMHO that doesn't seem like prime real estate we need to keep reserved. 

As best I can recall, I set it up as a user-level repo because I viewed it as experimental and wasn't confident at the time that I wouldn't need to wipe everything and start over.  But that didn't turn out to be necessary, so this temporary sandbox ended up being the accepted approach.  I think I also didn't have the required privileges to create/manage repos in the `pvlib` org at that time, but I'm not sure about that.
